### ISSUE TITLE
MusikCube crashes if an invalid FLAC gets loaded

### ISSUE THREAD

**Issue by PythonTryHard** [clangen/musikcube#328]
**Description**:
An invalid FLAC (0KB, complete devoid of all data) will still recognized by `musikcube` as a valid file and added as an entry in Libary. If the FLAC is loaded by directly playing, or loaded by gapless, `musikcube` will crash without producing any meaningful logs or error message.

**Steps to recreate**:
1. Create a text file without putting in any data.
2. Change the extension to FLAC and put it somewhere in the library directory (the more accessible the better)
3. Change `playback mode` to `gapless`
4. Play the FLAC, or the file before the FLAC (assuming no shuffle)

**Expected behavior**:
MusikCube not adding the file to library, or skip over the file when playback.

**Actual behavior**: 
Crash without message, log before crash only comes from `ffmpeg` reporting it cannot work with the file.

**Version affected**:
[v0.83 Windows 64-bit](https://github.com/clangen/musikcube/releases/download/0.83.0/musikcube_win64_0.83.0.zip)

**Note**:
Only FLACs have been tested.

**Comment by clangen** [comment]
Thanks for the report! This has been fixed for the next release.

**Comment by PythonTryHard** [comment]
I couldn't build `musikcube` on Windows somehow but I was able to build [commit 36ede666](https://github.com/clangen/musikcube/commit/36ede66603682fa3a72fc5d7dd1016f8a95af6b8) because I forgot the checkout on [b82e4b4](https://github.com/clangen/musikcube/commit/b82e4b4d652bd7401c09857b6bb01d1a7057f4db) on KDE Neon. Crash is confirmed to be fixed.

Closing issues, and looking forward to next release!
